### PR TITLE
Major release change

### PR DESCRIPTION
**Pull request by FrankIglesias**
## Summary
- Added clean bucket option
- Added output path option

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-deploy-script-fe",
-  "version": "0.2.0",
+  "version": "2.0.0",
```
Comment (damfinkel) [inline package.json]: This would be the first major I think, I'd put `1.0.0`

**Review by damfinkel — DISMISSED** [review]

File: script/s3.js [hunk 2]
```diff
@@ -39,28 +42,30 @@ const emptyBucket = (bucketName, callback) => {
     Bucket: bucketName
   };
 
-  uploader.listObjectsV2(params, function(err, data) {
+  uploader.listObjectsV2(params, function (err, data) {
```
Comment (mcallegari10) [inline script/s3.js]: Change to arrow function

File: script/s3.js [hunk 3]
```diff
@@ -69,14 +74,18 @@ const read = file => {
...
         {
           Bucket: credentials.bucket,
-          Key: file,
+          Key: fileKey,
           Body: base64data,
           ACL: "public-read",
-          CacheControl: "max-age=6048000",
-          Expires: 6048000,
+          CacheControl: CacheControl,
+          Expires: Expires,
```
Comment (mcallegari10) [inline script/s3.js]: Same names, no need to specify the variables 😄

File: script/s3.js [hunk 4]
```diff
@@ -91,15 +100,15 @@ const read = file => {
 
 const recursiveRead = (dir, done) => {
   var results = [];
-  fs.readdir(dir, function(err, list) {
+  fs.readdir(dir, function (err, list) {
```
Comment (mcallegari10) [inline script/s3.js]: Change to arrow function

File: script/s3.js [hunk 5]
```diff
@@ -91,15 +100,15 @@ const read = file => {
...
     if (err) return done(err);
     var pending = list.length;
     if (!pending) return done(null, results);
-    list.forEach(function(file) {
+    list.forEach(function (file) {
```
Comment (mcallegari10) [inline script/s3.js]: Same

File: script/s3.js [hunk 6]
```diff
@@ -91,15 +100,15 @@ const read = file => {
...
       file = path.resolve(dir, file);
-      fs.stat(file, function(_, stat) {
+      fs.stat(file, function (_, stat) {
```
Comment (mcallegari10) [inline script/s3.js]: Same

File: script/s3.js [hunk 7]
```diff
@@ -91,15 +100,15 @@ const read = file => {
...
         if (stat && stat.isDirectory()) {
-          recursiveRead(file, function(_, res) {
+          recursiveRead(file, function (_, res) {
```
Comment (mcallegari10) [inline script/s3.js]: Same

**Review by mcallegari10 — DISMISSED** [review]

**Pull request merged**